code/modules/hydroponics/eggincubator.dm

```diff
@@ -0,0 +1,108 @@
+/obj/machinery/egg_incubator
+	name = "egg incubator"
+	icon = 'icons/obj/virology.dmi'

```

Comment:
If you want to make it really shine, recolor it to  be more earthen with greens and browns.